doorbell-service/esp8266-nodemcu/init.lua

```diff
@@ -8,22 +8,12 @@ local BACKEND_SERVICE_URL = "http://<BACKEND_SERVICE_HOST>/api/chat.postMessage"
   "&link_names=1" ..
   "&text=%40here%3a+Someone%20is%20ringing%20the%20door%20bell!"
 
-function wait_until(condition, action)
-  local WAIT_TIME = 150
-  tmr.alarm(0, WAIT_TIME, 1, function()
-    if condition() then
-      tmr.stop(0)
-      action()
-    else
-      tmr.start(0)
-    end
-  end)
-end
-
 wifi.setmode(wifi.STATION)
 wifi.sta.config(WIFI_SSID, WIFI_PASSWORD)
```

Comment:
Same as above: I'd probably move this to after `eventMonStart` to avoid a possible race-condition.